Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityDiagnosticsSystemProfileInspector.cs

```diff
@@ -42,6 +43,11 @@ public override void OnInspectorGUI()
                 Selection.activeObject = MixedRealityManager.Instance.ActiveProfile;
             }
 
+            if (MixedRealityPreferences.LockProfiles && !((BaseMixedRealityProfile)target).IsCustomProfile)
```

Comment:
Should this go in the `MixedRealityBaseConfigurationProfileInspector` so we don't have to add it to each individual inspector?

Reply:
Ah, I see what you mean about the profile button. If there's a way to do it in the base, that'd be preferable, but it works as-is and that can be addressed later (and called out in the documentation).

Reply:
I tried to put it in the base initially, but it's not trivial.